Gemfile

```diff
@@ -64,9 +64,9 @@ gem 'pensions_calculator', '~> 1.4.3'
 gem 'quiz', '~> 1.2.0', source: 'http://gems.dev.mas.local'
 gem 'rio', '1.18.4', source: 'http://gems.dev.mas.local'
 gem 'savings_calculator', '~> 1.8.1'
-gem 'timelines', '~> 1.4.0'
-gem 'universal_credit', '2.16.0'
-gem 'wpcc', '1.11.10'
+gem 'timelines', '~> 1.5.0'
+gem 'universal_credit', '2.16.1'
+gem 'wpcc', '1.11.12'
```

Comment:
Why updating the version of this gems? They will influence the feature behaviour?

Reply:
It's a side effect to updating the MAS CMS Client gem to pick up changes. _Bundler_ did not permit me to run a straight `bundler install` to pick up the new changes nor `bundle update mas-cms-client`. 
Therefore, I had to run `bundle update`.

Reply:
I think it is best to return to the Gemfile to what it was before and tries again to update only mas-cms-client and then we found what is causing the bundle to fail. Otherwise you need to check what it changed between each version of each gem updated (timelines, universal_credit, wpcc). By the way, this branch needs rebase with master.